interface/client/appStart.js

```diff
@@ -24,7 +24,7 @@ mistInit = function(){
 
             Tabs.insert({
                 _id: 'browser',
-                url: 'https://ethereum.org',
+                url: 'http://dapps.ethercasts.com/',
```

Comment:
Unless there's some sort of legal issue on pointing to third party content, of course. I'll check it with Ming.